src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +26,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    <T> void fill(List<T> machines, T value);
```

Comment:
`List` - should be parameterized diffrently, 
Since we use parameterization at the class level, thiis `<T>` here is redundant.